applications/vanilla/modules/class.tagmodule.php

```diff
@@ -103,6 +103,13 @@ public function getData() {
 
         $this->autoContext();
 
+        $this->EventArguments['id'] = $this->ParentID;
```

Comment:
For the sake of simplicity, I'd keep these names the same. You'll know the ID maps to the ParentID of the tags and not an ID of a tag.